src/Orchestrator/appsettings.json

```diff
@@ -1,22 +1,22 @@
 {
   "Slice": {
-    "Hostname": ""
+    "Hostname": "Slice__Hostname"

```

Comment:
the values in here should be empty. I have filled them in, only to show you what should be the formatting of the env variable for each property.

Reply:
removed value